<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicNetworkTestWidget</class>
 <widget class="QWidget" name="MusicNetworkTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网速测试</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>475</width>
     <height>339</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>网速测试</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>345</y>
     <width>220</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>475</width>
     <height>314</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <widget class="MusicSpeedMeterWidget" name="speedWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="uploadSpeedValue">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orange</string>
   </property>
   <property name="text">
    <string>0Kbp/s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suspensionButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>悬浮框</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>284</y>
     <width>75</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>开始测速</string>
   </property>
  </widget>
  <widget class="QLabel" name="uploadSpeedLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>上行:</string>
   </property>
  </widget>
  <widget class="QLabel" name="downloadSpeedLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>105</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green</string>
   </property>
   <property name="text">
    <string>下行:</string>
   </property>
  </widget>
  <widget class="QLabel" name="downloadSpeedValue">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>105</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orange</string>
   </property>
   <property name="text">
    <string>0Kbp/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="uploadAllSpeedValue">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orange</string>
   </property>
   <property name="text">
    <string>0Kbp/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="uploadAllSpeedLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string>总上行:</string>
   </property>
  </widget>
  <widget class="QLabel" name="downloadAllSpeedValue">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>175</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orange</string>
   </property>
   <property name="text">
    <string>0Kbp/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="downloadAllSpeedLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>175</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green</string>
   </property>
   <property name="text">
    <string>总下行:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>网卡设置</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>loadingLabel</zorder>
  <zorder>speedWidget</zorder>
  <zorder>uploadSpeedValue</zorder>
  <zorder>suspensionButton</zorder>
  <zorder>testButton</zorder>
  <zorder>uploadSpeedLabel</zorder>
  <zorder>downloadSpeedLabel</zorder>
  <zorder>downloadSpeedValue</zorder>
  <zorder>uploadAllSpeedValue</zorder>
  <zorder>uploadAllSpeedLabel</zorder>
  <zorder>downloadAllSpeedLabel</zorder>
  <zorder>downloadAllSpeedValue</zorder>
  <zorder>settingButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicSpeedMeterWidget</class>
   <extends>QWidget</extends>
   <header>musicspeedmeterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
